javascript/ql/src/semmle/javascript/security/dataflow/CleartextLogging.qll

```diff
@@ -42,6 +42,24 @@ module CleartextLogging {
         write.getRhs() = src and
         trg.(DataFlow::SourceNode).flowsTo(write.getBase())
       )
+      or
+      // Taint step through `util.inspect(..)` from Node.js
+      trg = DataFlow::moduleImport("util").getAMethodCall("inspect") and
```

Comment:
It's better to use `getAMemberCall` or `DataFlow::moduleMember("util", "inspect").getACall()`.

`getAMethodCall` won't catch cases like
```javascript
import { inspect } from 'util';
inspect(...)
```